recipes/ccache/all/conanfile.py

```diff
@@ -79,35 +89,43 @@ def package_id(self):
         del self.info.settings.compiler
 
     def build_requirements(self):
-        if hasattr(self, "settings_build") and tools.cross_building(self) and \
+        self.build_requires("pkgconf/1.9.3")
```

Comment:
Why need pkgconfig now?

Reply:
For PkgConfigDeps - CMake aborts with an error if its not installed

Reply:
Please, use CMakeDeps instead, do not need pkgconfig.